.github/workflows/ci.yml

```diff
@@ -29,7 +29,6 @@ jobs:
         - '2.5'
         - jruby
         - '2.4'
-        - '2.3'
```

Comment:
this won't work, you need to add `.github/workflows/custom.ci` and include just this node there, it'll be merged into the final ci config via devtools GH action.

Reply:
fixed, but that way it makes current GH actions for 2.3 fail

Reply:
@IvanShamatov I'll remove 2.3 from required builds